Bottom: d4c5d11eb16c4b407b723f5e18f0b567974618c2
Top:    d24b2b5392b102194b4cf8c9aaf99c432fe68238
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 22:38:35 +0000

Refresh of attempt-to-cache-emission-of

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index b1ab93a..f40613c 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -385,39 +385,36 @@ object3d_from_contours (POLYAREA *contours,
 
   /* Loop over all board outline pieces */
   pa = contours;
-<<<<<<< current
   do
     {
-=======
-  do {
 
-    contour = pa->contours;
-    ncontours = 0;
-    npoints = 0;
+      contour = pa->contours;
+      ncontours = 0;
+      npoints = 0;
 
-    ct = contour;
-    while (ct != NULL) {
-      ncontours ++;
-      npoints += get_contour_npoints (ct);
-      ct = ct->next;
-    }
+      ct = contour;
+      while (ct != NULL) {
+        ncontours ++;
+        npoints += get_contour_npoints (ct);
+        ct = ct->next;
+      }
 
-    object = make_object3d (PCB->Name);
+      object = make_object3d (PCB->Name);
 
 #if 0
-    /* XXX: REF-COUNTING WOULD BE WAY BETTER! */
-    if (master_object_appearance != NULL)
-      {
-        object_appearance = make_appearance ();
-        appearance_set_appearance (object_appearance, master_object_appearance);
-      }
+      /* XXX: REF-COUNTING WOULD BE WAY BETTER! */
+      if (master_object_appearance != NULL)
+        {
+          object_appearance = make_appearance ();
+          appearance_set_appearance (object_appearance, master_object_appearance);
+        }
 
-    /* XXX: REF-COUNTING WOULD BE WAY BETTER! */
-    if (master_top_bot_appearance != NULL)
-      {
-        top_bot_appearance = make_appearance ();
-        appearance_set_appearance (top_bot_appearance, master_top_bot_appearance);
-      }
+      /* XXX: REF-COUNTING WOULD BE WAY BETTER! */
+      if (master_top_bot_appearance != NULL)
+        {
+          top_bot_appearance = make_appearance ();
+          appearance_set_appearance (top_bot_appearance, master_top_bot_appearance);
+        }
 #else
 #endif
     object_appearance = master_object_appearance;
